wp-content/themes/oceanwp-child/assets/css/template-styles.css

```diff
@@ -90,13 +106,16 @@ h4{
     
 }
 
-.initiative-tags{
+.tags{
```

Comment:
Are tags used outside initiatives? Same tags as in "news"?

Reply:
Yes